components/sections/app_promo/variant_b.tsx

```diff
@@ -25,102 +30,60 @@ function VariantB({
-                      className="order-0 md:order-0 lg:order-0 mx-2 hidden rounded-full bg-white p-2 text-webriq-darkblue shadow hover:text-webriq-babyblue focus:outline-none sm:inline-block md:mr-12 md:p-4 lg:mr-12 lg:p-4 xl:order-1 xl:mr-12 xl:p-4 2xl:order-1 2xl:mr-12 2xl:p-4"
-                      onClick={arrowLeftClick}
-                    >
-                      <svg
-                        className="h-6 w-6"
-                        xmlns="http://www.w3.org/2000/svg"
-                        fill="none"
-                        viewBox="0 0 24 24"
-                        stroke="currentColor"
-                      >
-                        <path
-                          strokeLinecap="round"
-                          strokeLinejoin="round"
-                          strokeWidth={2}
-                          d="M10 19l-7-7m0 0l7-7m-7 7h18"
-                        />
-                      </svg>
-                    </button>
-                    <div className="xl:order-0 2xl:order-0 order-1 mr-2 w-1/2 object-contain md:order-1 lg:order-1">
-                      {images?.[currentPosition]?.image && (
-                        <Image
-                          className="mx-auto mb-8 w-full object-cover xl:mb-0"
-                          src={urlFor(images?.[currentPosition]?.image)}
-                          sizes="100vw"
-                          width={500}
-                          height={850}
-                          alt={
-                            images?.[currentPosition]?.alt ??
-                            `appPromo-variantB-image${currentPosition}`
-                          }
-                        />
-                      )}
-                    </div>
-                    <button
-                      aria-label="Right Arrow button"
-                      className="order-2 hidden rounded-full bg-white p-2 text-webriq-darkblue shadow hover:text-webriq-babyblue focus:outline-none sm:inline-block md:ml-12 md:p-4 lg:ml-12 lg:p-4 xl:ml-12 xl:p-4 2xl:ml-12 2xl:p-4"
-                      onClick={arrowRightClick}
-                    >
-                      <svg
-                        className="h-6 w-6"
-                        xmlns="http://www.w3.org/2000/svg"
-                        fill="none"
-                        viewBox="0 0 24 24"
-                        stroke="currentColor"
-                      >
-                        <path
-                          strokeLinecap="round"
-                          strokeLinejoin="round"
-                          strokeWidth={2}
-                          d="M14 5l7 7m0 0l-7 7m7-7H3"
-                        />
-                      </svg>
-                    </button>
-                  </>
-                ) : (
-                  <div className="object-contain md:w-2/5 xl:w-2/5">
+    <section className="py-20 bg-gray-50">
+      <Container>
+        <Flex wrap>
+          <div className="w-full mt-8 lg:w-1/2">
+            <Text
```

Comment:
Kindly update the Subtitle, Title and Description elements to be rendered only when it is defined/there is a value.